benchmark.py

```diff
@@ -346,6 +347,13 @@ def pop_outputs_thread(output_q):
         default="Fp16_b",
         help="Set data format",
     )
+    parser.add_argument(
+        "-adf",
+        "--acc_dataformat",
+        choices=["Fp32", "Fp16", "Fp16_b", "Bfp8", "Bfp8_b", "Bfp4", "Bfp4_b"],
+        default="Fp32",
```

Comment:
I'm assuming this wont have an impact on running Grayskull since there is no Fp32 support there?

Reply:
Yes, this shouldn't impact Grayskull. However, it might be better to not set the default accumulation data format, but rather use it only when defined in benchmark test run

Reply:
I just converted this one for option. This way it's much more flexible with current and future architectures.